src/Model/RequestManager.php

```diff
@@ -77,4 +78,15 @@ public function insert(Request $request): bool
 
         return $result;
     }
+
+    public function updateOnAnswerer(int $anwererId, int $requestId): bool
+    {
+        $statement = $this->pdo->prepare("UPDATE " . self::TABLE .
+        " SET fk_answerer_id=:answererId" . " WHERE request.id=:requestId");
+        $statement->bindValue('answererId', $anwererId, \PDO::PARAM_INT);
+        $statement->bindValue('requestId', $requestId, \PDO::PARAM_INT);
+        $result = $statement->execute();
+
+        return $result;
```

Comment:
Database checks are missing

Reply:
The check is done in the controller. Our method return a bool val and we check if it's true or not

Reply:
But what if PDO::prepare fails !
It could if the database is in a bad state.